data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -85,7 +86,8 @@
     (remove_records_with_wrong_date.get_remove_records_with_wrong_date_queries,
     ),
     (invalid_procedure_source.get_remove_invalid_procedure_source_queries,),
-    (CleanMappingExtTables,)
+    (CleanMappingExtTables,),
+    (remove_ehr_data.remove_ehr_data_queries,),
```

Comment:
I think this needs to be above `CleanMappingExtTables`, I had recently added a cleaning rule in DEID_BASE and lauren had the following comment:

(CleanMappingExtTables,), needs to be the last rule run in this list. It's purpose is to drop any mapping records that no longer have a record in the OMOP tables.

Reply:
I changed the location!